libs/external-db-security/src/services/role_authorization.ts

```diff
@@ -1,41 +1,44 @@
+import { CollectionPermissions, VeloRole, WixDataRole } from "@wix-velo/velo-external-db-types"
+
 const { errors } = require('@wix-velo/velo-external-db-commons')
 const { UnauthorizedError } = errors
 const DefaultPolicies = ['Admin']
 
-class RoleAuthorizationService {
-    constructor(config) {
-        this.config = config || []
+export default class RoleAuthorizationService {
+    config: CollectionPermissions[]
+    constructor(config: CollectionPermissions[] = []) {
+        this.config = config
     }
 
-    authorizeRead(collectionName, role) {
+    authorizeRead(collectionName: string, role: WixDataRole) {
         const readPolicies = this.readPoliciesFor(collectionName)
 
         if (!readPolicies.includes(this.dataToVeloRole(role))) throw new UnauthorizedError('You are not authorized')
     }
     
-    authorizeWrite(collectionName, role) {
+    authorizeWrite(collectionName: string, role: WixDataRole) {
         const writePolicies = this.writePoliciesFor(collectionName)
         
         if (!writePolicies.includes(this.dataToVeloRole(role))) throw new UnauthorizedError('You are not authorized')
     }
 
-    setConfig(config) {
+    setConfig(config: CollectionPermissions[]) {
         this.config = config
     }
 
-    policiesFor(collectionName) {
+    policiesFor(collectionName: string) {
         return this.config.find(p => p.id === collectionName)
     }
 
-    readPoliciesFor(collectionName) {
+    readPoliciesFor(collectionName: string) {
         return this.policiesFor(collectionName)?.read || DefaultPolicies
     }
 
-    writePoliciesFor(collectionName) {
+    writePoliciesFor(collectionName: string) {
         return this.policiesFor(collectionName)?.write || DefaultPolicies
     }
 
-    dataToVeloRole(role) { 
+    dataToVeloRole(role: WixDataRole): VeloRole { 
```

Comment:
Why do you need to specify exactly what result is returned from the function? TS can not deduce this alone?